### PR TITLE
OCP4: Write rule to check audit log forwarding uses TLS

### PR DESCRIPTION
**Pull request by JAORMX**
This ensures that TLS is being used as an output for all defined outputs
in the ClusterLogForwarder instance.

Signed-off-by: Juan Antonio Osorio Robles <jaosorior@redhat.com>

### CODE REVIEW

**Comment by JAORMX** [comment]
Pull-request updated, HEAD is now d805dd1c9c187bdb193faa7c301b8ddc22492109

**Comment by JAORMX** [comment]
Pull-request updated, HEAD is now 9069f7b1daf3bd46c4008cee2629ae1eea020d37

**Review by jhrozek — APPROVED** [review]
LGTM

File: applications/openshift/logging/audit_log_forwarding_uses_tls/rule.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+rationale: |-
+  It is necessary to ensure that any configured output uses the TLS protocol to receive
+  logs in order to ensure the confidentiality, integrity and authenticity of the logs.
+
+references:
+  nist: AU-9,AU-9(2),AU-9(3),AU-10
+
+identifiers:
+  cce@ocp4: CCE-90688-3
```
Comment (jhrozek) [inline applications/openshift/logging/audit_log_forwarding_uses_tls/rule.yml:26]: Wait a sec, where does the CCE come from? Did you maybe forget to remove a line from the CCE list?

**Review by jhrozek — CHANGES_REQUESTED** [review]
question about the CCE otherwise looks good

File: applications/openshift/logging/audit_log_forwarding_uses_tls/rule.yml (hunk 1 above)
Comment (JAORMX) [reply applications/openshift/logging/audit_log_forwarding_uses_tls/rule.yml:26]: sure did. Let me fix this

**Comment by JAORMX** [comment]
Pull-request updated, HEAD is now af5b72833c2fae54c996c1fb5307ea6a4e66f87d

**Comment by JAORMX** [comment]
@jhrozek could you revisit this?

**Review by jhrozek — APPROVED** [review]
LGTM
looking good!

**Pull request merged**